README.md

```diff
@@ -7,29 +7,34 @@ Node.js tools for Visual Studio 2017 is developed and managed here.
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+<!--and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
+-->
 
 Feel free to file issues or ask questions on our [issue tracker](http://github.com/Microsoft/nodejstools/issues), 
 and we welcome code contributions - see 
 [Contributing](https://github.com/Microsoft/nodejstools/blob/master/.github/CONTRIBUTING.md) for information. 
 
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
 The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
 and Enterprise. 
 To install, run the [normal VS installer](https://visualstudio.com/vs/downloads?wt.mc_id=github_microsoft_com) 
 and select the **Node.js development workload**.
 
-For the latest update to the Node.js tools install [Visual Studio Preview](https://www.visualstudio.com/vs/preview/).
+If you already have Visual Studio and only need to download the Node.js Tools, you can fine them [here](https://visualstudio.microsoft.com/downloads/?wt_mc_id=github_microsoft_com)
```

Comment:
Is this true? Node.js Tools should be installable as a workload on the Visual Studio Installer. I think we can point the user how to install the workload instead.

Reply:
Also, the link doesn't seem to point to anything related to Node.js

Reply:
nope youre totally right!.. I read the description on them wrong

Reply:
removing!